app/controllers/minio_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "securerandom"
+
+# This controller handles file uploads to minio buckets
+# specifically for uploading images for e.g. POIs
+class MinioController < ApplicationController
+
+  before_action :verify_current_user
+  before_action :minio_setup
+
+  def signed_url
+    bucket = @minio_config["bucket"]
+    headers = {}
+    options = { path_style: true }
+
+    url = @storage.put_object_url(
+      bucket,
+      generate_filename,
+      15.minutes.from_now.to_time.to_i,
```

Comment:
was ist in 15 Minuten?

Reply:
Ich vermute mal, dass die signed_object_url in 15 minutes expired

https://github.com/fog/fog-aws/blob/master/lib/fog/aws/requests/storage/put_object_url.rb